extension/src/popup/components/manageNetwork/NetworkForm/index.tsx

```diff
@@ -127,25 +300,35 @@ export const NetworkForm = ({ isEditing }: NetworkFormProps) => {
               </Field>
             )}
             {isEditing ? (
-              <Button
-                disabled={!(isValid && dirty)}
-                fullWidth
-                isLoading={isSubmitting}
-                type="submit"
-                variant={Button.variant.tertiary}
-              >
-                {t("Add network")}
-              </Button>
+              <div className="NetworkForm__editing-buttons">
+                <Button
+                  onClick={() => history.goBack()}
+                  type="button"
+                  variant={Button.variant.tertiary}
+                  fullWidth
+                >
+                  {t("Cancel")}
+                </Button>
+                <Button
+                  disabled={!isValid || isMainnetOrTestnet}
+                  isLoading={isSubmitting}
+                  fullWidth
+                  type="submit"
+                >
+                  {t("Save")}
+                </Button>
+              </div>
             ) : (
-              <Button
-                disabled={!(isValid && dirty)}
-                fullWidth
-                isLoading={isSubmitting}
-                type="submit"
-                variant={Button.variant.tertiary}
-              >
-                {t("Save")}
-              </Button>
+              <div className="NetworkForm__add-button">
+                <Button
+                  disabled={!(isValid && dirty) || isMainnetOrTestnet}
```

Comment:
doesn't this line block adding a new network if you're currently on Mainnet or testnet? I just tried it locally and I wasn't able to add a new network

Reply:
Ugh, you're right - this was the last thing I added and I didn't test it enough

Reply:
Just addressed this. There was a def a bug. Also, `isMainnetOrTestnet` actually referred to whether you were trying to edit those configs, not if you were currently using it. I renamed that var to make it more clear